plugins/chat/test/javascripts/acceptance/user-status-on-mentions-test.js

```diff
@@ -355,30 +369,6 @@ acceptance("Chat | User status on mentions", function (needs) {
     await click(".chat-composer-button.-send");
   }
 
-  function setupAutocompleteResponses(results) {
```

Comment:
I wonder why have you inlined this function?

Reply:
It was used in a single place, which was like:

```
before {
  7 pretenders
  thatFunction()
}

thatFunction() {
  2 pretenders
}
```

I don't think there was any value in extracting those two to a function (we could add a comment `// Autocomplete responses` to the same effect)